main.go

```diff
@@ -29,7 +29,7 @@ var (
 	version     = "dev"
 	commit      = "unknown"
 	date        = "unknown"
-	specVersion = "0.10.1"
+	specVersion = "0.11.0-dev"
```

Comment:
Is it correct to be `0.11.0-dev` and not `0.11.0`?

Reply:
Good catch - because this is now the final PR, I'll bump it to be 0.11.0.